src/pages/career.tsx

```diff
@@ -1,29 +1,138 @@
+import Layout from '../components/layout/layout';
+import { theme } from '../components/layout/theme';
+import { up } from '../components/breakpoint/breakpoint';
+import {
+  PageTitle,
+  LargeText,
+  Text,
+  SubTitle as TypoSubTitle,
+} from '../components/typography/typography';
 import { Grid, GridItem } from '../components/grid/grid';
-import { Card } from '../components/cards/card';
+import { CareerGrid } from '../components/card-grids/career-grid';
+
+const SubTitle = styled(TypoSubTitle)`
+  margin: 80px 0 40px;
+
+  ${up('md')} {
+    margin: 160px 0 40px;
+  }
+`;
+
+const Note = styled.div`
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 110%;
+  letter-spacing: -0.15px;
+  color: #202840;
+`;
+
+const StyledText = styled(Text)`
+  &:last-of-type {
+    margin-bottom: 0;
+  }
+`;
+
+const StyledLink = styled(Link)`
+  color: ${theme.palette.text.darkLinkColor.default};
+  text-decoration: none;
+
+  &:hover {
+    color: ${theme.palette.text.darkLinkColor.hover};
+  }
+`;
 
 const CareerPage: React.FC = () => {
   return (
     <Layout>
       <SEO title="Career" />
-      <Grid>
-        <GridItem>
+      <Grid center>
+        <GridItem md={8}>
           <PageTitle>Career</PageTitle>
+          <LargeText>
+            It’s not a job you do, it’s a career your choose.
+          </LargeText>
+          <StyledText>
+            Satellytes bewirbt sich bei dir. Wir sind Geeks, die das was sie
+            tun, richtig gut und gerne tun, aber noch viel lieber
+            weitervermitteln. Denn zusammen schaffen wir mehr. Wir wollen mit
+            dir arbeiten – egal ob Junior oder Senior – und unser Wissen und
+            unsere Jahrzehnte an Erfahrung an dich weitergeben. Das Wichtigste
+            was du mitbringen musst: Neugier und Leidenschaft für die Sache —
+            den Rest machen wir zusammen.
+          </StyledText>
+          <StyledText>
+            Bewirb dich jetzt bei uns als UI- und/oder UX-Designer, Frontend-
+            oder Backend-Developer. Deine Ausbildung ist sekundär, wenn du eine
+            schnelle Auffassungsgabe hast, sympathisch bist und einfach richtig
+            Bock hast.
+          </StyledText>
         </GridItem>
       </Grid>
-      <Grid>
-        <Card
-          title="Junior UI Designer"
-          text="Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Maecenas sed diam eget risus varius blandit sit amet non magna."
-          link="/"
-        />
+      <Grid center>
+        <GridItem xs={12}>
+          <SubTitle as="h2">Opportunities</SubTitle>
+        </GridItem>
+
+        <GridItem xs={12}>
+          <CareerGrid careerData={careerData} />
+        </GridItem>
+
+        <GridItem xs={12}>
+          <Note>
+            Oder schicke deine Bewerbung an:{' '}
+            <StyledLink to="mailto:team@satellytes.com">
```

Comment:
think this needs to be a standard a tag/external link, because isn't Gatsby Link only for internal page links?

Reply:
🙏
I took the mailto out for now, as we actually don't know what is expected here & since mailto isn't necessarily best practise.